README.rst

```diff
@@ -106,6 +106,25 @@ Optional
 .. |netlistsvg| replace:: ``netlistsvg``
 .. _netlistsvg: https://github.com/nturley/netlistsvg
 
+* |ghdl|_
+
+.. |ghdl| replace:: ``ghdl``
+.. _ghdl: https://github.com/ghdl/ghdl
+
+GHDL and ghdl-yosys-plugin are required for VHDL support. If GHDL is prebuilt into Yosys,
+add this configuration option to let Yosys know::
+
+    hdl_diagram_ghdl = "prebuilt"
+
+Otherwise, to load GHDL as a runtime module, set this configuration option to::
+
+    hdl_diagram_ghdl = "module"
+
+Which will pass ``-m ghdl`` to Yosys when calling it. Similarly, setting this to the path of a
```

Comment:
Note that this is kind of hackish. We are relying on Yosys guessing that `ghdl` is, in fact, the name of a shared library (a module) that is to be found in some predefined location. In the containers we create either in ghdl/docker or hdl/containers, we do guarantee that. However, users installing ghdl-yosys-plugin on their own, might have it located anywhere. In those cases, `hdl_diagram_ghdl` should support a `path/to/module.[so|dll]` value.

See https://github.com/YosysHQ/yosys/issues/2408#issuecomment-739136153 and https://github.com/ghdl/ghdl-yosys-plugin#build-as-a-module-shared-library.

So, you explained it properly here, but it's slightly misleading in the docs (below).